Cubical/HITs/EilenbergMacLane1/Properties.agda

```diff
@@ -0,0 +1,116 @@
+{-
+
+Eilenberg–Mac Lane type K(G, 1)
+
+-}
+
+{-# OPTIONS --cubical --no-import-sorts --safe #-}
+module Cubical.HITs.EilenbergMacLane1.Properties where
+
+open import Cubical.HITs.EilenbergMacLane1.Base
+
+open import Cubical.Core.Everything
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Isomorphism
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.HLevels
+
+open import Cubical.Data.Sigma
+
+open import Cubical.Algebra.Group.Base
+
+open import Cubical.HITs.PropositionalTruncation as PropTrunc using (∥_∥; ∣_∣; squash)
+open import Cubical.HITs.SetTruncation as SetTrunc using (∥_∥₂; ∣_∣₂; squash₂)
+
+private
+  variable
+    ℓG ℓ : Level
+
+module _ ((G , str) : Group {ℓG}) where
+
+  open GroupStr str
+
+  elimEq : {B : EM₁ (G , str) → Type ℓ}
+           (Bprop : (x : EM₁ (G , str)) → isProp (B x))
+           {x y : EM₁ (G , str)}
+           (eq : x ≡ y)
+           (bx : B x)
+           (by : B y) →
+           PathP (λ i → B (eq i)) bx by
+  elimEq {B = B} Bprop {x = x} =
+    J (λ y eq → ∀ bx by → PathP (λ i → B (eq i)) bx by) (λ bx by → Bprop x bx by)
```

Comment:
This is an instance of a general lemma:
```
  elimEq Bprop eq = isProp→PathP (λ i → Bprop (eq i))
```
Is it worth it to give this special case a name? I noticed now that a similar lemma is defined in `SetQuotients.Properties` and `GroupoidQuotients.Properties`, but it doesn't look like they're used except to prove the `elimProp` equivalent...

Reply:
Inlined everywhere now..

Reply:
Done